end_to_end_tests/baseline_openapi_3.0.json

```diff
@@ -2858,56 +2858,6 @@
       "ModelWithBackslashInDescription": {
         "type": "object",
         "description": "Description with special character: \\"
-      },
```

Comment:
Here and in the next file, and in `golden_record`, I've removed some schemas that no longer need to be in the golden-record-based tests. There are now better equivalent tests in `test_unions.py`.